frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -46,44 +46,41 @@ export const StyledContentList = styled('ol')(({ theme }) => ({
     '& > li + li': {
         borderTop: `1px solid ${theme.palette.divider}`,
     },
-    '&:not(li > &) > li:first-of-type': {
-        // select first list elements in lists that are not directly nested
-        // within other lists.
+    '& > li:first-of-type': {
         paddingTop: theme.spacing(1),
     },
-    '& > li:has(> ol)': {
-        // nested lists add their own padding to their list items, so we don't want to double it up.
-        paddingBlock: 0,
-    },
 }));
 
-export const StyledListItem = styled('li', {
-    shouldForwardProp: (prop) => prop !== 'type',
-})<{ type?: 'release plan' | 'strategy' }>(({ theme, type }) => ({
-    background:
-        type === 'release plan'
-            ? theme.palette.background.elevation2
-            : theme.palette.background.elevation1,
-}));
+const releasePlanBackground = (theme: Theme) =>
+    theme.palette.background.elevation2;
+const strategyBackground = (theme: Theme) =>
+    theme.palette.background.elevation1;
 
-const PaginatedStrategyContainer = styled('div')(({ theme }) => ({
-    paddingTop: theme.spacing(2.5),
-    position: 'relative',
-    display: 'flex',
-    flexFlow: 'column nowrap',
-    gap: theme.spacing(2),
+export const StyledListItem = styled('li')<{
+    'data-type'?: 'release-plan' | 'strategy';
```

Comment:
With the data-type attribute, we can pass it down to html and target it with css.